core-bundle/src/Resources/config/services.yml

```diff
@@ -815,3 +815,15 @@ services:
             - '@request_stack'
             - '@contao.framework'
             - '@contao.routing.scope_matcher'
+
+    contao.util.simple_token_parser:
```

Comment:
We decided to start using the class name for services. This will also enable autowiring for this class.

Reply:
Did we really? I thought we would only use this for Controllers and Helpers where we want to use autowiring ourselves - but I may confuse something? 

I thought using the class name in a bundle is [bad practice](https://symfony.com/doc/current/bundles/best_practices.html#services). We could introduce an interface and use this instead (+ make the class final) but imo that's not worth the overhead?